### PR TITLE
theme.yamal fixes, typo fixes, re did shop page using flexboxes, …

### PR DESCRIPTION
**Pull request by Mopok**
fixed featured items on home page. This is first 10 or so checkboxes on Ross's fix list seen here:
https://github.com/lemonstand/lemonstand-2/issues/3760

### CODE REVIEW

File: pages/registered/page-registered.htm [hunk 1]
```diff
@@ -25,6 +26,6 @@
 <div class="jumbotron row featured text-center">
 	<h4 class="buffer">Continue Shopping or Shop our featured collection</h4>
```
Comment (tomcornall) [inline pages/registered/page-registered.htm]: Could be useful to link to the "/shop", like this:

`<h4 class="buffer"><a href="/shop">Continue Shopping</a> or Shop from our featured collection</h4>`

File: partials/home-featured-products.htm [hunk 2]
```diff
@@ -1,11 +1,14 @@
 <div class="col-md-8 col-md-offset-2 pricing-box">
-{% if featuredCollection %}
-
-  
-  {% for product in featuredCollection.products %}
+	{% for product in featuredCollection.products %}
+		{{ partial('shop-grid-item-home', {'product':product}) }}
+	{% else %}
```
Comment (tomcornall) [inline partials/home-featured-products.htm:4]: 👍  Good fallback here

File: partials/shop-product-loop.htm [hunk 3]
```diff
@@ -1,16 +1,17 @@
 {% set shop_item = theme.shopItemsPerPage %}
```
Comment (tomcornall) [inline partials/shop-product-loop.htm]: Can probably remove this first set of `shop_item`, it's getting overwritten in the next line anyways.

File: theme.yaml [hunk 4]
```diff
@@ -7,17 +7,18 @@ engine: twig
 404: '404'
 customFields:
     storeTitle: { type: text, title: 'Title of your Store', default: Coffree }
+    featuredSubscriptions: { type: text, title: 'Name of your featured subscription plans', default: featuredPlans }
```
Comment (tomcornall) [inline theme.yaml]: Should be something like:

"Name of your subscription product collection"

File: resources/css/combined.css [hunk 5]
```diff
@@ -434,11 +434,69 @@ sub{
...
+	width: 100%;
+}
+.shop-list-content {
+	display: flex;
+	flex-direction: column;
+}
+.shop-product-description {
+	flex: 1 0 auto;
+}
+
```
Comment (tomcornall) [inline resources/css/combined.css:450]: extra line

File: resources/css/combined.css [hunk 6]
```diff
@@ -434,11 +434,69 @@ sub{
...
+	background-image: none;
+	border: 1px solid #494c50;
+	border-radius: 4px;
+}
+.pagination > ul > li.active > a {
+	background: #dedede;
 }
 .cat-list{
 	padding-left:50px;
+	padding-top:50px;
```
Comment (tomcornall) [inline resources/css/combined.css]: Needs a space between the colon and the value

File: resources/css/combined.css [hunk 7]
```diff
@@ -7,19 +7,19 @@ body{
...
 }
 .jumbotron{
-	max-width:1444px;
-	margin: 0 auto;
+	
 	background:white;
 
 	border-bottom:1px solid #f1f1f1;
-	padding:70px 0;
+
```
Comment (tomcornall) [inline resources/css/combined.css]: Some weird spacing in this .jumbotron class (realize a lot of this was already here): should remove extra lines, and needs space between colons and values

File: resources/css/combined.css [hunk 8]
```diff
@@ -7,19 +7,19 @@ body{
...
 	border-bottom:1px solid #f1f1f1;
-	padding:70px 0;
+	
+}
+.row {
+	margin-left: 0px;
+	margin-right: 0px; 
 }
 .jumbotron p{
 	font-weight:400;
```
Comment (tomcornall) [inline resources/css/combined.css]: another space needed, b/w colon and value

**Review by tomcornall — CHANGES_REQUESTED** [review]
@Mopok All very minor feedback, looks good.

- [x] Need screenshot of Pagination after centering.

**Comment by Mopok** [comment]
Showed centred pagination
screen shot 2017-02-17 at 10 45 15 am

**Review by tomcornall — APPROVED** [review]

**Pull request merged**